enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/catchup/storecopy/FileSender.java

```diff
@@ -46,7 +46,7 @@
     FileSender( StoreResource resource )
     {
         this.resource = resource;
-        this.byteBuffer = ByteBuffer.allocateDirect( MAX_SIZE );
+        this.byteBuffer = ByteBuffer.allocate( MAX_SIZE );
```

Comment:
Might this not cause extra copying of the buffer contents when transferring data between network and storage?